.evergreen/run-tests.sh

```diff
@@ -12,6 +12,7 @@ AUTH=${AUTH:-noauth}
 SSL=${SSL:-nossl}
 MONGODB_URI=${MONGODB_URI:-}
 TESTS=${TESTS:-}
+export API_VERSION=${API_VERSION:-}
```

Comment:
`API_VERSION` is used in the tests, while the others are used in this script, hence the `export` to avoid having to modify the call to `phpunit` below.

Reply:
`MONGODB_URI` is also used by PHPUnit and it doesn't have an `export` line.

Reply:
Hm, no idea why I added it since it also works without.